aptos-move/framework/cached-packages/build.rs

```diff
@@ -2,20 +2,49 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use framework::ReleaseTarget;
+use std::env::current_dir;
 use std::path::PathBuf;
 
 fn main() {
     // Set the below variable to skip the building step. This might be useful if the build
     // is broken so it can be debugged with the old outdated artifacts.
     if std::env::var("SKIP_FRAMEWORK_BUILD").is_err() {
-        println!("cargo:rerun-if-changed=../aptos-token/sources");
-        println!("cargo:rerun-if-changed=../aptos-token/Move.toml");
-        println!("cargo:rerun-if-changed=../aptos-framework/sources");
-        println!("cargo:rerun-if-changed=../aptos-framework/Move.toml");
-        println!("cargo:rerun-if-changed=../aptos-stdlib/sources");
-        println!("cargo:rerun-if-changed=../aptos-stdlib/Move.toml");
-        println!("cargo:rerun-if-changed=../move-stdlib/sources");
-        println!("cargo:rerun-if-changed=../move-stdlib/Move.toml");
+        let current_dir = current_dir().expect("Should be able to get current dir");
+        // Get the previous directory
+        let mut prev_dir = current_dir;
+        prev_dir.pop();
```

Comment:
Have we tested this running from root? Not sure if it just pops to root again, I assume so.

Reply:
Yeah root pop becomes root.  Pop returns a book on whether it could pop